config.tutorial.yaml

```diff
@@ -6,9 +6,10 @@ version: 0.4.2
 tutorial: true
 
 
-countries: ["NG", "BJ"]
+countries: ["NG"]
```

Comment:
Please, restore BJ, is there a reason for the proposed change?

Reply:
Thanks @davide-f, I cant sincerely remember if there was an issue with this. But I will revert BJ

Reply:
no problem, great :D